src/CryptSync.cpp

```diff
@@ -89,6 +89,7 @@ int APIENTRY _tWinMain(HINSTANCE hInstance,
                              L"/useGPG       : use GnuPG encryption\n"
                              L"/fat          : use FAT write time accuracy (2s)\n"
                              L"/nosyncdeleted: don't sync file deletions\n"
+                             L"/resetoriarchattr: reset archive attribute on original file\n" 
```

Comment:
the name `resetoriarchattr `is a little bit confusing. Maybe just name it `resetarchiveattr `or `cleararchiveattr `?

Reply:
Changed to resetarchiveattr